google/cloud/testing_util/example_driver.cc

```diff
@@ -61,10 +61,14 @@ int Example::Run(int argc, char const* const argv[]) try {
 
   std::string const command_name = argv[1];
   auto command = commands_.find(command_name);
-  if (commands_.end() == command) {
+  if (commands_.end() == command && command_name != "--help") {
     PrintUsage(argv[0], "Unknown command: " + command_name);
     return 1;
   }
+  else if(commands_.end() == command && command_name == "--help") {
+    PrintUsage(argv[0], "");
```

Comment:
Aside: It looks like a number of `testing_util::Example` users do not handle "--help" in their commands (by throwing a `testing_util::Usage`), so `full_usage_` is not going to be complete.

Reply:
Sorry about not being clearer.  Indeed, I don't think there is anything actionable for you here.  It is more a reminder for us that any user of `testing_util::Example` is expected to handle "--help" for all their non-auto commands.